crates/veryl/src/runner/cocotb.rs

```diff
@@ -163,6 +163,16 @@ impl Runner for Cocotb {
 
         let module = format!("{}_{}", metadata.project.name, top.unwrap());
 
+        let (py_waves, args) = if let Some(WaveFormFormat::Vcd) = wave {
```

Comment:
For adding another waveform format in future, using `match` instead of `if let` may be better.